Purchases/Purchasing/Purchases.swift

```diff
@@ -1331,25 +1331,28 @@ public extension Purchases {
     }
 #endif
 
+// todo: obsolete checkPromotionalDiscountEligibility
```

Comment:
this is done in `Obsoletions.swift` I think?

Reply:
Oops sorry, I thought I removed this 🙊